cmd/run.go

```diff
@@ -191,7 +224,11 @@ var RunCmd = &cobra.Command{
 				PID:          os.Getpid(),
 			})
 
-			print.SuccessStatusEvent(os.Stdout, "You're up and running! Both Dapr and your app logs will appear here.\n")
+			if output.AppCMD != nil {
+				print.SuccessStatusEvent(os.Stdout, "You're up and running! Both Dapr and your app logs will appear here.\n")
+			} else {
+				print.SuccessStatusEvent(os.Stdout, "You're up and running! Dapr logs will appear here.\n")
```

Comment:
This output makes the fact that Dapr is running in "sidecar only" mode a little subtle here.  Perhaps something more explicit in case the user simply forgot to add the application command (which I have done from time to time).  (Also a argument for an explicit "sidecar only" switch.)

Reply:
I've added a warning in bold white in the beginning of the run.

If we add a switch, then we need to document the case where use passes the switch and the application command. Users might get confused the other way around: way is my app not running? Oh, I forgot to remove the switch. I think we can avoid a new switch for this use case. The warning is the first thing users will see as the output for this command.